eng/pipelines/source-build-license-scan.yml

```diff
@@ -0,0 +1,154 @@
+# This pipeline runs the license scan test for the VMR (https://github.com/dotnet/dotnet).
```

Comment:
From a developer guidance perspective, it would be nice if the documentation include a direct link to the pipeline to run.  Having that within the pipeline yml feels a bit off though.

Reply:
What's the difference where the link is? IMO, having it in a separate markdown file doesn't make a difference as far as maintenance goes. I feel that having the doc in the pipeline YAML itself is better because it's closer to the source that it's describing.

Reply:
In general if we consider this developer documentation having a MD file is more convenient for the reader IMO.  I don't have strong feeling about this other than a direct link be provided to the AzDO pipeline.

Reply:
Fixed in https://github.com/dotnet/installer/pull/17442/commits/ad71a5a747a56e400e70e7291623362e99dfbc21